src/Microsoft.AspNet.BasicMiddleware/HttpMethodOverrideExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Builder;
+
+namespace Microsoft.AspNet.BasicMiddleware
```

Comment:
This class should go in the Builder namespace. For example, look at the extension methods for all the other middlesware: https://github.com/aspnet/Mvc/blob/faaba481e82f15e83b6ff02241fe2c55e568a9d0/src/Microsoft.AspNet.Mvc.Core/BuilderExtensions.cs#L11